docs/azure-monitor-logging.md

```diff
@@ -11,64 +11,50 @@ A few built-in Azure Monitor fields are overloaded:
     "transport": "MQTT"
   }
 ```
 
-## Fields always available for device and service apps
-
-| field | format | meaning |
-| - | - | - |
-| `sdkLanguageVersion` | string | see same variable in [metrics.md](./metrics.md) |
-| `sdkLanguage` | string | see same variable in [metrics.md](./metrics.md) |
-| `sdkVersion` | string | see same variable in [metrics.md](./metrics.md) |
-| `osType` | string | see same variable in [metrics.md](./metrics.md) |
-| `poolId` | string | name of service app pool being used |
-| `runId` | guid | `runId` for app generating metrics |
-| `hub` | string | name of hub being used for testing |
-
-## Fields available only for device apps
-
-| field | format | meaning |
-| - | - | - |
-| `transport` | string | transport being used, if available (may not be available on service apps).  One of `mqtt`, `mqttws`, `amqp`, or `amqpws` |
-
-## Fields available for device and maybe available for service app.
-
-Service app will include these fields if they are available, but not all service app tracing has these available.  For example, global service app setup is not specific to a device ID, so it doesn't include the `deviceId` field.
-
-| field | format | meaning |
-| - | - | - |
-| `deviceId` | string | device ID being used. |
-| `pairingId` | guid | pairing ID for device and service relationshiop. |
-
-## Automatically populated fields which are meaningless for us
-
-| field | format | meaning |
-| - | - | - |
-| `sessionId` | string | populated by python Azure Monitor wrappers. |
-| `process` | string | populated by Python Azure Monitor wrappers. |
+| field | format | device/service | meaning |
+| - | - | - | - |
+| `sdkLanguageVersion` | string | both | see same variable in [metrics.md](./metrics.md) |
+| `sdkLanguage` | string | both | see same variable in [metrics.md](./metrics.md) |
+| `sdkVersion` | string | both | see same variable in [metrics.md](./metrics.md) |
+| `osType` | string | both | see same variable in [metrics.md](./metrics.md) |
+| `poolId` | string | both | name of service pool being used |
+| `hub` | string | both | name of hub being used for testing |
+| `transport` | string | device | transport being used, if available One of `mqtt`, `mqttws`, `amqp`, or `amqpws` |
+| `deviceId` | string | both. |  device ID being used. Some service logs don't contain this. |
+| `runId` | string | both |  guid representing the run of the device app for this message. Some service logs don't contain this. |
+| `serviceInstance` | string | service |  guid representing the run of the service app for this message. (service apps only). |
+
+### Notes on deviceId and RunId
+Some service app logs are device specific and contain `deviceId` and `runId` values.
+Other service app logs are not device specific and do not contain `deviceId` and `runId` values.
 
 ## Automaticly populated fields for Python traces.
 
+```json
+  {
+    "lineNumber": "291",
+    "fileName": "service.py",
+    "level": "INFO",
```

Comment:
Will this always match the level of the trace in AppInsights?

Reply:
Good question.  This is done by the OpenCensus libraries that we use to link the python logging infrastructure to AI.  I suspect there is a "best effort" mapping but it's probably not perfect, but I haven't dug into this yet.